website/more_documentation/button_documentation.py

```diff
@@ -33,3 +38,21 @@ async def await_button_click() -> None:
             ui.label('Two')
             await b.clicked()
             ui.label('Three')
+
+    @text_demo('Disable button with a context manager', '''
+        This showcases a async context manager that can be used to disable a button for the duration of an async process.
+    ''')
+    async def disable_context_manager() -> None:
+        @asynccontextmanager
+        async def disable(element: DisableableElement) -> None:
```

Comment:
Why are you using an _async_ context manager and an async `disable` function? I think we can use the synchronous versions here.

Reply:
I'm new to async and probably just ended up there incorrectly with something working. I tried it with a normal context manager and it seems to give the same behavior, so I switched to that.